python/pyspark/mllib/classification.py

```diff
@@ -428,9 +428,10 @@ def save(self, sc, path):
     def load(cls, sc, path):
         java_model = sc._jvm.org.apache.spark.mllib.classification.NaiveBayesModel.load(
             sc._jsc.sc(), path)
-        py_labels = _java2py(sc, java_model.labels())
-        py_pi = _java2py(sc, java_model.pi())
-        py_theta = _java2py(sc, java_model.theta())
+        # Can not unpickle array.array from Pyrolite in Python3 with "bytes"
```

Comment:
Will this change depending on upstream Pyrolite patches?  If so, can we create a JIRA and add a `TODO` referencing it so that we know which changes to roll back when that happens?